src/Components/test/testassets/ComponentsApp.Server/Pages/Index.cshtml

```diff
@@ -14,6 +14,15 @@
 <body>
     <app>@(await Html.RenderComponentAsync<App>())</app>
 
+    <script type="text/javascript">
+        window.Blazor = {
+            callbacks: {
+                configureSignalR: function (builder) {
+                    builder.configureLogging(0);
+                }
+            }
+        }
+    </script>
```

Comment:
A few things to note:

This has to happen before `components.server.js` so that it can be used to configure signalR - we create the connection immediately when `components.server.js` is included.

I made this a callback on `signalR.HubConnectionBuilder` because there might be more things on the signal r connection that are useful. Then we won't have to add more API for dealing with signal r.

I made `window.Blazor.callbacks` its own object so we can treat it as opaque see `GlobalExports.ts`. 

`configureLogging` takes a .ts enum - however we don't have the signal r stuff exported from our script. I think we we end up using this pattern we will want to resolve this somehow, possibly by signal r allowing a string.